src/compas_rhino/geometry/brep/edge.py

```diff
@@ -62,17 +71,19 @@ def _set_edge(self, rhino_edge):
 
     @property
     def data(self):
-        curve_type, curve = self._get_curve_geometry()
+        curve_type, curve, plane, domain = self._get_curve_geometry()
         return {
             "curve_type": curve_type,
             "curve": curve.data,
```

Comment:
in the future this will go away, i guess, when we add live objects instead of their data...